n2v/grid/grider.py

```diff
@@ -291,9 +297,9 @@ def on_grid_density(self, grid=None,
             points_function.compute_points(i_block)
             b_points = i_block.npoints()
             offset += b_points
-            density[offset - b_points : offset, 0] = 0.5 * rho_a.np[ :b_points]
+            density[offset - b_points : offset, 0] = rho_a.np[ :b_points]
```

Comment:
The original cube file generator does not distinguish between restricted and unrestricted, but they always have the 0.5 factor. I thought I did some testing and the factor was appropriate. Was there anything that you found incorrect with that?

Reply:
Ye. I think with 0.5 it seems incorrect. 
e.g. (This is the result from my current code without 0.5)
#F atom
n = iF.on_grid_density(vpot=iF.Vpot)
w = iF.Vpot.get_np_xyzw()[-1]
n.shape
print(np.sum(n[:,0]*w), np.sum(n[:,-1]*w))
==>
4.999999999999997 4.000000000000006

Reply:
Yup. This is right. Thanks for finding it. Not really important for plotting a density. But huge for calculations.